x-pack/plugins/enterprise_search/public/applications/app.test.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+
+import { coreMock } from 'src/core/public/mocks';
+import { mountWithIntl } from 'test_utils/enzyme_helpers';
```

Comment:
ooh, super interesting. I assume this is something Kibana ships with?

Although I'm curious - we're not using `Intl`, so we shouldn't _need_ to mount with it (although presumably we will implement intl/i18n at some point).

Would a regular old enzyme `mount()` work instead or are there other intervening layers in this `mountWithIntl` that's making this test work?

Reply:
Yes, absolutely, it's not necessary. I mainly chose to keep it like this 'cause this is how other projects are mounting things, and I figured if we ever implement i18n it'll just work. If it bothers you we can remove it, I have no strong opinions.

Reply:
Yeah I figured you'd grabbed it from another plugin's tests. I'm good with it personally. From my past experience writing Enzyme mount helpers, it's _possible_ we'll start running into extra hoops / needing to dive an extra amount of times past the intl HOC, but we can reevaluate if that starts happening.

Reply:
That's true. On second though, I might just end up removing it in my next PR.